ext/curl/interface.c

```diff
@@ -2229,13 +2233,22 @@ static inline int build_mime_structure_from_hash(php_curl *ch, zval *zpostfields
 				cb_arg->filename = zend_string_copy(postval);
 				cb_arg->stream = NULL;
 
+				if ((stream = php_stream_open_wrapper(ZSTR_VAL(postval), "rb", STREAM_MUST_SEEK, NULL))) {
+					if (!php_stream_stat(stream, &ssb)) {
```

Comment:
```suggestion
					if (!stream->readfilters.head && !php_stream_stat(stream, &ssb)) {
```

stat() tends to be unreliable when stream filters are involved.

Reply:
Ah, thanks, haven't thought about stream filters.